codegen/src/main/java/software/amazon/awssdk/codegen/poet/client/AsyncClientClass.java

```diff
@@ -128,6 +129,14 @@ private MethodSpec closeMethod() {
 
     }
 
+    @Override
+    protected MethodSpec.Builder paginatedTraditionalMethodBody(MethodSpec.Builder builder, OperationModel opModel) {
```

Comment:
What does traditional mean in this context? Non-simple?

Reply:
Yep, a method that takes in a request and returns a response.

Reply:
Got it. There is no need for the "traditional" part as its the only method body for paginated operations. Will remove.